File: app/src/main/java/dev/ragnarok/fenrir/api/model/longpoll/MessageFlagsSetUpdate.java
package dev.ragnarok.fenrir.api.model.longpoll;

public class MessageFlagsSetUpdate extends AbsLongpollEvent {

    //[[2,1030891,128,216143660]]

    public int message_id;
    public int mask;
    public int peer_id;

    public MessageFlagsSetUpdate() {
        super(ACTION_MESSAGES_FLAGS_SET);
    }

    public int getMessageId() {
        return message_id;
    }

    public int getPeerId() {
        return peer_id;
    }

    public int getMask() {
        return mask;
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/api/model/longpoll/MessageFlagsSetUpdate.kt
package dev.ragnarok.fenrir.api.model.longpoll

class MessageFlagsSetUpdate :
    AbsLongpollEvent(ACTION_MESSAGES_FLAGS_SET) {
    //[[2,1030891,128,216143660]]
    var messageId = 0
    var mask = 0
    var peerId = 0
}